site/docs/components/tokenized-input-next/usage.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+### Using the component
+
+The component is typically used in a form. It’s ideal for fields that can contain multiple, specific values related to a single topic or context—and these values can be removed quickly and in their entirety. You would use the component, for example, in a field for email addresses or a fast field for a cash ticket.
+
+### When not to use
+
+- As a standalone component. It will fail to meet ADA requirements.
```

Comment:
Is this due to the labelling?

Reply:
this just came from the old docs, removing it